AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -16,7 +16,7 @@ import Amplify
 public class AmplifyAWSServiceConfiguration {
 
     /// - Tag: AmplifyAWSServiceConfiguration.amplifyVersion
-    public static let amplifyVersion = "2.12.0"
+    public static let amplifyVersion = "visionos-preview"
```

Comment:
no need of a number? something like `visionos-preview-0.0.1`, will be useful in keeping track of bug fixes.

Reply:
Good call. Addressed in [e5e51ed](https://github.com/aws-amplify/amplify-swift/pull/3058/commits/e5e51eda6ef435d295c822df2bcd401238a472c1).